inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/knowledge_base.adoc

```diff
@@ -0,0 +1,62 @@
+[[sect_knowledge_base]]
+=== Knowledge Base 
+
+The Knowledge base (KB) module of INCEpTION enables the user to create a KB from scratch or to import it from an RDF file. Alternatively, the user can connect to a remote KB using SPARQL. However, editing the content of remote KBs is currently not supported. The Knowledge base enables the user to annotate the text with entities from KB. 
+
+This section briefly describes how to set up a KB in the KB management page on *Projects Settings* and explain the features provided on the *Knowledge Base* page along with the support for feature layers providing the annotations for KB entities. 
+
+==== Setting up a knowledge base 
+
+In the *Projects Settings*, switch to the *Knowledge Bases* tab, then click *New…* on the bottom
+ and a dialogue box shows as in the figure below.
+
+[.thumb]
+image::kb1.png[align="center"]
+
+To create a *LOCAL | REMOTE*  knowledge base, one needs to choose *Local* or *Remote* for the type. For the reification,
+*NONE* is the default case, but to support qualifiers, one needs to choose *WIKIDATA* for the Reification. 
+
+For the local KB, the user has to choose a file, in case one needs to import a local KB. Alternatively, the user can skip the step to create an empty KB to create a knowledge base from scratch. 
+
+For the remote KB, INCEpTION provides the user with some pre-configured knowledge base such as WikiData, British Museum, BabelNet, DBPediaa and Yago. The user has an option to set up a remote KB *apart* from the pre-configured ones, in which case the user needs to provide the SPARQL endpoint URL for the Knowledge base as in the figure below.
+
+[.thumb]
+image::kb2.png[align="center"]
+
+In the next step, the user is offered a configurable mapping which relies on a minimal set of IRIs that must be defined for any KB used with the platform. The user can choose one of the pre-configured mapping or provide their own custom defined mapping as shown in the figure below. In the advanced settings, the user can leverage this feature of KB settings when one doesn't want the entire knowledge base to be used and rather choose to identify some specific root concepts. This feature specially helps in case of large knowledge bases such as Wikidata.
+ 
+[.thumb]
+image::kb3.png[align="center"]
+
+==== Knowledge Base Page
+
+The knowledge base page provides you with a concept tree hierarchy with a list of instances and statements associated along with the list of properties as shown in the figure below. The user can edit the *LOCAL* knowledge base here which includes adding, editing and deletion of concepts, properties, statements and instances. 
```

Comment:
provides a concept tree hierarchy with a list of instances and statements,  together with the list of properties as shown in the figure below. Here the user can edit the *LOCAL* knowledge base, which includes adding, editing and deleting concepts, properties, statements and instances.